code/modules/antagonists/cult/blood_magic.dm

```diff
@@ -141,14 +141,14 @@
 	name = "Stun"
 	desc = "Empowers your hand to stun and mute a victim on contact."
 	button_icon_state = "hand"
-	magic_path = "/obj/item/melee/blood_magic/stun"
+	magic_path = /obj/item/melee/blood_magic/stun
```

Comment:
Does this mean there's a `text2path()` somewhere that can be removed?

Reply:
oh yea probably

Reply:
actually no, previously it was doing new "/obj/item/melee/blood_magic/stun", which is a no no in the coding policy thing